ch1-basic/ch1-03-array-string-and-slice.md

```diff
@@ -221,7 +221,7 @@ type  SliceHeader  struct {
 ```
 
 Slice được xem là fat pointer, các bạn có thể đọc thêm ở bài viết sau để hiểu hơn về fat pointer trong [Go](https://nullprogram.com/blog/2019/06/30/). Cấu trúc slice bao gồm:
-- `Data`: là con trỏ chứa địa chỉ của một array.
+- `Data`: chứa giá của con trỏ tới một array.
```

Comment:
nit: "chứa đỉa chỉ trong bộ nhớ của con trỏ trỏ tới underlying array của slice"

Reply:
It is clear. Tks bro.